File: poc-challenges/250724/src/test/java/com/tdd/domain/FibonacciTest.java
package com.tdd.domain;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class FibonacciTest {
	@Test
	void fibonacci_test() {
		int[][] numbers = {{0,0}, {1,1}, {1,2}, {2,3}, {3,4}, {5,5}, {8,6}, {13,7}, {21,8}};

		for (int[] number : numbers) {
			assertEquals(number[0], Fibonacci.fib(number[1]));
		}
	}
}

File: poc-challenges/250912/src/test/kotlin/com/currenjin/tdd/FibonacciTest.kt
package com.currenjin.tdd

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class FibonacciTest {
    @Test
    fun fib() {
        assertEquals(0, Fibonacci.fib(0))
        assertEquals(1, Fibonacci.fib(1))
    }
}